### ISSUE TITLE
Splash screen autohide does not work when application rootview is an external url

### ISSUE THREAD

**Issue by pre** [AppGyver/steroids#821]
Supersonic currently injects a script tag which calls `supersonic.app.splashscreen.hide()`. When application is started in an external url, this script tag cannot be injected. As a result splash screen is not hidden.

By default Supersonic disables autohide of the splashscreen to overcome loading issues of Web Components  on Android.

Autohide is defined in `config.xml` by the property `  <preference name="AutoHideSplashScreen" value="true" />`

**Comment by seanconormcdonnell** [comment]
have there been any updates?

**Comment by harsaharsa** [comment]
Nope, still missing a native solution to the issue (which is linked to https://github.com/AppGyver/steroids/issues/830 – the splashscreen autohide hack was implemented to circumvent this on app load).

A workaround is to run a local clone of Steroids and change https://github.com/AppGyver/steroids/blob/master/src/steroids/features.coffee#L6 to be false, after which you can write

```
splashscreen:
  autohide: true
```

in your `config/app.coffee`.
